src/figuresClasses.ts

```diff
@@ -1,19 +1,107 @@
+// type Shape = 'triangle' | 'circle' | 'rectangle';
+
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+// type Color = 'red' | 'green' | 'blue';
+
+enum Color {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue',
+}
+
 export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
 
+function normalizeArea(areaValue: number): number {
+  return Math.trunc(areaValue * 100) / 100;
 }
 
-export class Triangle {
+const ERROR_MESSAGE = 'Value cannot be <= 0!';
```

Comment:
But would be good also to give more descriptive name for this variable(what type of error message it is?)